adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -271,6 +281,7 @@ public void export(ExtraFileFoundNotice toExport) throws IOException {
         protoBuilder.setCsvFileName(toExport.getFilename())
                 .setType(GtfsValidationOutputProto.GtfsProblem.Type.TYPE_UNKNOWN_FILE)
                 .setSeverity(GtfsValidationOutputProto.GtfsProblem.Severity.WARNING)
+                .clearAltEntityId()
```

Comment:
Good catch! As we said together, replace this by a call to `clear()` at the beginning of the chain to clear the reused internal builder